fix: fix and update the tutorials for `0.17` (#744)
cosmos/sdk-tutorials: blog-legacy/tutorial/05-starport-type.md

Running this command will create and add all the core functionality for the type `comment`, including registering entrypoints in `rest/` and `cli/`, as well as defining the relevant `types`, `handler`, `messages`, and `keeper`.

> Once this is done, run `starport chain serve` and it will start up your backend server, the consensus engine, and a frontend user interface. This information is available when you run your application and visit `localhost:12345`.

Reviewer: ```suggestion
After the core functionality for the type `comment` is added, run `starport chain serve` to start your backend server, the consensus engine, and a frontend user interface. This information is available when you run your application and visit `localhost:12345`.
```

Reviewer: instead of the unclear reference "this" let's specify the subject 
from the Google developer documentation style guide

> this, that
> Where possible, put a noun after this or that for clarity. If doing so results in clunky prose, then don't do it; but even then, try thinking about what the noun would be. If you aren't sure what noun this or that refers to, then consider rephrasing.
